smslibrary/src/main/java/com/eis/communication/Peer.java

```diff
@@ -0,0 +1,21 @@
+package com.eis.communication;
+
+import androidx.annotation.NonNull;
+
+import java.io.Serializable;
+
+/**
+ * Represents a user in the network
+ * @param <A> The type of address used by the network
+ */
+public interface Peer<A> extends Serializable {
+
+    A getAddress();
+
+    void setAddress(A address);
```

Comment:
Missing specifications for this methods (even though the name makes it obvious)

Reply:
Holy cow! I forgot to add them, thanks.